### DISCUSSION TITLE
New FBA Inbound API v2024-03-20  - Placement options Q&A on items returned

### DISCUSSION THREAD

**Discussion by MetaMan2023** [amzn/selling-partner-api-models#4397] [Q&A]
Implementing new FBA API as there is a looming deadline.  I have some questions:

* The system returns multiple placement options for us -- for example 3-5 different choices (with various "splits").   Has there been any discussion on allowing us to specific how many splits we want up front, possibly in an attempt to speed up the process? (would it?)   If for example I know I only want to send to the East Coast 2 centers, it would be nice if I could just say that up front.   NOTE: The only reason to do this would be if it saved significant amount of time, for example if it cut the time down 50-75%.

* We notice it offers multiple choices for 1 center splits -- for example one east Coast, one middle, one west coast with wildly different Inbound Placement Fee options... (almost double from one coast to the other). 

Questions:
--  Why does it even over west coast options when I'm on the east coast if the fees are so much higher? 
-- Could it ever be that the fee to the opposite coast would be lower?
-- Would there be a reason I would want to send to the other coast? 
-- The only thing I can come up with is when there were hurricane's in Florida, we redirected to the central zone to avoid having inventory issues/delays.

* We notice that, even to the East Coast, the IPF varies when comparing 1 split to 2 -- what I mean by that is I may send one set of products and the IPF difference (per unit) between 1 and 2 centers could be 15 cents -- and on another shipment it could be 2 cents.   Is that due to the size of the products or is Amazon offering a discount based on center capacity?  

* We notice that even if we offer 5 identical boxes, we may not get the option to send to 5 centers with no IPF fee.. for example If I have 5 identical packed boxes and then 3 more that are a different size (EXAMPLE: 10, 10, 10, 10, 10, 12, 12, 12) that I am not offered the 5 split no fee option.   Is the API requiring me to have ALL my boxes be identical?  Or is it that, while I have 5 x 10 unit boxes, I also need 5 x 12 unit boxes?    This example is a subtle gray area on the 5 identical box rule.

**Comment by puppsupr** [comment]
Hi,
Thanks for the feedback. Look below for the answers,

Amazon offers three placement options: Minimal Shipment Splits (send to fewest centers for a fee) and Partial or Optimized Splits (send to multiple locations with reduced or no fee). Availability depends on factors like item type, quantity, inventory levels, and customer demand. Shipments may be split for reasons such as specialized items, safety handling, FBA Label Service processing, or to optimize delivery times. 

APIs don't have control over the fees and splits and its at the program level. For more information on placement options refer [here](https://seller-central-master.amazon.com/help/hub/reference/GC3Q44PBK8BXQW3Z).

For FBA program level questions, please contact Seller Support.

**Reply by MetaMan2023** [reply]
@puppsupr 

Regarding the "5 center / no fee" option -- as long as our inbounding boxes meet compliance (ie: 5 or more identically packed boxes), should we technically always be offered that option?  Or are you saying that Amazon may not offer the 5 center split (no IPF) option based on customer demand/inventory levels?
